supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -251,7 +251,7 @@ For example, instead of "Red Hat recommends using X package because", write "Use
 [discrete]
 [[red-hat-container-catalog]]
 ==== image:images/no.png[no] Red Hat Container Catalog (noun)
-*Description*: "Red Hat Container Catalog" was the Red Hat-hosted registry for enterprise-ready containers located at link:https://access.redhat.com/containers[access.redhat.com/containers].
+*Description*: _Red Hat Container Catalog_ was the Red Hat-hosted registry for enterprise-ready containers located at link:https://access.redhat.com/containers[access.redhat.com/containers].
```

Comment:
Link works, but redirects to catalog.redhat.com - so you might want to change so that when the redirect stops working the link works.

Reply:
Updated.